post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -0,0 +1,262 @@
+2. **Configure Amazon Simple Email Servie**
+    - Verify the Email address you are going to use for sending the Email notifications
+3. **Create a custom policy**
+
+    <details>
+    <summary>Using the AWS console</summary>
+
+    1. Go to **Services > IAM**.
+    2. On the left, click **Policies**.
+    3. In the main pane, click **Create policy**.
+    4. Click the **JSON** tab.
+    5. Paste the [JSON code below](#JSON) into the text box.
+    6. Click **Review policy**.
+    7. On the **Review policy** page:
+        - In the **Name** field, enter a name. Example: `FSS_SES_Send_Email_Policy`.
+        - Click **Create policy**.
+    8. Click the link to your policy to open its summary.
+    9. Take note of the **Policy ARN** near the top of the page.
+    </details>
+
+    <details>
+    <summary>Using the AWS CLI</summary>
+
+    1. Paste the [JSON code below](#JSON) into a file called `fss-ses-send-email-policy.json` (or another name).
+    2. In a shell program such as bash or Windows Powershell, enter the following AWS CLI command to create the policy:
+
+        `aws iam create-policy --policy-name <YOUR_FSS_SES_SEND_EMAIL_POLICY> --policy-document file://fss-ses-send-email-policy.json`
+
+        where `<YOUR_FSS_SES_SEND_EMAIL_POLICY>` is replaced with the name you want to give to the custom policy. Example: `FSS_SES_Send_Email_Policy`.
+    3. In the output, take note of the custom policy's ARN. Example: `arn:aws:iam::0123456789012:policy/FSS_SES_Send_Email_Policy`
+    </details>
+
+    <details>
+    <summary><a name="JSON">JSON code (for use in the custom policy)</a></summary>
+
+    ```json
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "VisualEditor0",
+                    "Effect": "Allow",
+                    "Action": [
+                        "ses:SendEmail",
+                        "ses:SendRawEmail"
+                    ],
+                    "Resource": "*"
+                }
+            ]
+        }
+    ```
+
+    </details>
+
+4. **Create an execution role for the Lambda function**
+
+    <details>
+    <summary>Using the AWS console</summary>
+
+    1. Go to **Services > IAM**.
+    2. Click **Roles** on the left.
+    3. In the main pane, click **Create role**.
+    4. Under **Select type of trusted entity**:
+        - Select the **AWS service** box.
+        - Click the  **Lambda** service from the list.
+        - Click **Next: Permissions**.
+    5. In the search box:
+        - Search for `AWSLambdaBasicExecutionRole`.
+        - Select its check box.
+        - Search for `<YOUR_FSS_SES_SEND_EMAIL_POLICY>` where `<YOUR_FSS_SES_SEND_EMAIL_POLICY>` is the name of your custom File Storage Security policy. Example: `FSS_SES_Send_Email_Policy`.
+        - Select its check box in the list.
+        - You now have two policies selected.
```

Comment:
You now have two policies selected.

Reply:
I dont't understand that comment.

Reply:
This shouldn't start with a hyphen because it's not an active step that the user takes. It's just the result of the previous step, so it shouldn't be on the same level as the steps.

Reply:
Changed